articles/quickstart/native/ionic2/01-login.md

```diff
@@ -10,72 +10,85 @@ budicon: 448
   repo: 'auth0-ionic2-samples',
   path: '01-Login',
   requirements: [
-    'Ionic 2.0.0-rc.0',
-    'Angular 2+'
+    'Ionic 3.x',
+    'Angular 4+'
   ]
 }) %>
 
-## Set Up the Callback URL
+## Overview
+
+To integrate Auth0 in a native Ionic app, you can use the `@auth0/cordova` package available on npm. This package provides an interface with cordova which allows you to use the [Proof Key for Code Exchange](https://tools.ietf.org/html/rfc7636) spec. PKCE is recommended for native applications to mitigate the threat of authorization code interception.
+
+## Set Up the Client
 
 <div class="setup-callback">
-<p>Go to the <a href="${manage_url}/#/applications/${account.clientId}/settings">Application Settings</a> section in the Auth0 dashboard and make sure that <b>Allowed Callback URLs</b> contains the following value:</p>
+<p>Go to the <a href="${manage_url}/#/applications/${account.clientId}/settings">Application Settings</a> section in the Auth0 dashboard and make sure that the <b>Allowed Callback URLs</b> contains the following value (replace `YOUR_PACKAGE_ID` with your app identifier):</p>
 
-<pre><code>https://${account.namespace}/mobile</pre></code>
+<pre><code>YOUR_PACKAGE_ID://${account.namespace}/cordova/YOUR_PACKAGE_ID/callback</pre></code>
```

Comment:
Can we add a note on where/how package IDs are configured? It's possible that some Ionic devs won't have ever thought about their package ID yet. Let's add this in to the readme of the sample too where it makes sense.